src/UI/Components/Diagram/Canvas.tsx

```diff
@@ -318,100 +349,49 @@ const Canvas: React.FC<{
-                selected.model as ServiceModel,
-                selected.name === mainServiceName,
-                selected.isEmbedded,
-                selected.holderName,
-              );
-              shape.set("sanitizedAttrs", sanitizedAttrs);
-              handleUpdate(shape, ActionEnum.CREATE);
             }
           }
         }}
       />
       <Toolbar
-        openEntityModal={() => {
-          setIsFormModalOpen(true);
-        }}
-        serviceName={mainServiceName}
+        serviceName={mainService.name}
         handleDeploy={handleDeploy}
         isDeployDisabled={
           instancesToSend.size < 1 || !isDirty || looseEmbedded.size > 0
         }
         editable={editable}
       />
       <CanvasWrapper id="canvas-wrapper" data-testid="Composer-Container">
-        <div className="canvas" ref={canvas} />
-        <ZoomWrapper>
-          <button
-            className="zoom-in"
-            onClick={(event) => {
-              event.stopPropagation();
-
-              if (diagramHandlers) {
-                diagramHandlers.zoom(1);
-              }
-            }}
-          >
-            +
-          </button>
-          <button
-            className="zoom-out"
-            onClick={(event) => {
-              event.stopPropagation();
-              if (diagramHandlers) {
-                diagramHandlers.zoom(-1);
-              }
-            }}
-          >
-            -
-          </button>
-        </ZoomWrapper>
+        <StencilContainer className="stencil-sidebar" ref={LeftSidebar} />
+        <div className="canvas" ref={Canvas} />
+        <ZoomHandlerWrapper className="zoomHandler" ref={ZoomHandler} />
       </CanvasWrapper>
     </>
   );
 };
-export default Canvas;
 
-const ZoomWrapper = styled.div`
-  display: flex;
-  gap: 1px;
+/**
+ * Wrapper and ref container for the zoom & fullscreen tools from JointJS
+ */
+const ZoomHandlerWrapper = styled.div`
   position: absolute;
   bottom: 16px;
   right: 16px;
-  box-shadow: 0px 4px 4px 0px
-    var(--pf-v5-global--BackgroundColor--dark-transparent-200);
-  border-radius: 5px;
-  background: var(--pf-v5-global--BackgroundColor--dark-transparent-200);
-
-  button {
-    display: flex;
-    width: 24px;
-    height: 22px;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    flex-shrink: 0;
-    background: var(--pf-v5-global--BackgroundColor--100);
-    padding: 0;
-    border: 0;
+`;
 
-    &.zoom-in {
-      border-top-left-radius: 4px;
-      border-bottom-left-radius: 4px;
-    }
-    &.zoom-out {
-      border-top-right-radius: 4px;
-      border-bottom-right-radius: 4px;
-    }
-    &:hover {
-      background: var(--pf-v5-global--BackgroundColor--light-300);
-    }
-    &:active {
-      background: var(--pf-v5-global--Color--light-300);
-    }
-  }
+/**
+ * To be able have draggables on the canvas, we need to have a stencil container to which we append the JointJS stencil objects that handle this scenario
```

Comment:
```suggestion
 * To be able to have draggable items on the canvas, we need to have a stencil container to which we append the JointJS stencil objects that handle this scenario
```

I'd be more concrete than saying 'this scenario' 
in six months, we will be wondering which scenario we are talking about.